Makefile

```diff
@@ -134,6 +134,28 @@ doc: $(GLOBFILES) $(VFILES)
 	--with-header $(ENHANCEDDOCSOURCE)/header.html $(VFILES)
 	sed -i'.bk' -f $(ENHANCEDDOCSOURCE)/proofs-toggle.sed $(ENHANCEDDOCTARGET)/*html
 
+# Jason Gross' coq-tools bug isolator:
+# Override this on the command line; this is just an example:
```

Comment:
Oh, I see that that is also part of PR #266 . Is this PR superseded completely by #266 ?